anchor/network/src/peer_manager/peerdb/client.rs

```diff
@@ -0,0 +1,201 @@
+//! Known Ethereum 2.0 clients and their fingerprints.
+//!
+//! Currently using identify to fingerprint.
+
+use libp2p::identify::Info as IdentifyInfo;
+use serde::Serialize;
+use strum::{AsRefStr, EnumIter, IntoStaticStr};
+
+/// Various client and protocol information related to a node.
+#[derive(Clone, Debug, Serialize)]
+pub struct Client {
+    /// The client's name (Ex: lighthouse, prism, nimbus, etc)
+    pub kind: ClientKind,
+    /// The client's version.
+    pub version: String,
+    /// The OS version of the client.
+    pub os_version: String,
+    /// The libp2p protocol version.
+    pub protocol_version: String,
+    /// Identify agent string
+    pub agent_string: Option<String>,
+}
+
+#[derive(Clone, Copy, Debug, Serialize, PartialEq, AsRefStr, IntoStaticStr, EnumIter)]
+pub enum ClientKind {
```

Comment:
We probably should change this just to have Anchor and SSV. 

I think they are using libp2p `identify`, so once we figure out their agent string, we can identify them here.